### DISCUSSION TITLE
请教NAS公网ip修改问题

### DISCUSSION THREAD

**Discussion by Zhigang27** [jeessy2/ddns-go#911] [General]
我把DDNS-GO部署在群辉NAS上，总体来说还比较好用。我目前在IPV4那里的domain name里填写的是：www.xiaowu.net。当我的公网IP发生变化的时候，主机记录为www的确实可以修改成新的ip。

那么请问，主机记录为@的这个解析怎么修改？

注：我的邮箱也是部署在群辉NAS上，用Synology MailPlus的。域名是在阿里云上的。

**Comment by jeessy2** [comment]
不填@。会自动处理的
example.com

**Reply by Zhigang27** [reply]
您写的example.com，意思是我不用前面添加www对吗？我之前是写的www.xiaowu.net。

我前面是发现他没有把@记录自动修改过来，不知道啥原因；那，如果有mail.xiaowu.net，这个需要填写在domain name里吗？

**Reply by jeessy2** [reply]
example.com和www.example.com是二个域名

这些你自己看到操作，多操作就熟悉了

**Reply by Zhigang27** [reply]
好吧，那我还是把：
@:xiaowu.net
www.xiaowu.net
mail.xiaowu.net
这三个都列上，再观察一阵子看看。因为我看到DDNS-GO页面上写的：一行一个域名，可使用冒号手动分隔主机记录，如：ddns:example.com.cn

我的理解是冒号前面的ddns就是主机记录，请问这样理解对吗？

**Reply by jeessy2** [reply]
冒号是针对不能程序自动分割的，你那个域名不用

**Comment by Zhigang27** [comment]
你看下面两个截图左上角的时间，第一个是自动的，第二个是我今天发现邮件不太正常了，我自己去阿里云看域名，发现@这个记录没有修改。我之前在DDNS-GO里写的是www.xiaowu.net。但只修改了www这个主机记录。是不是应该就只写xiaowu.net，不要前面的www？

<img width="692" alt="截屏2023-11-11 19 52 16" src="https://github.com/jeessy2/ddns-go/assets/118618521/e88ee5e7-904c-4707-a3a3-713113b1c3de">
<img width="681" alt="截屏2023-11-11 19 52 28" src="https://github.com/jeessy2/ddns-go/assets/118618521/b85815a5-1276-44b5-9a05-7460a0183029">

**Reply by jeessy2** [reply]
不要

**Reply by Zhigang27** [reply]
好的，谢谢。另外，我还有一个阿里云的账号，邮箱用的也是群辉上的，但网站是找百度帮忙做的。那，如果才能只修改这个域名的邮件有关的解析，而不去动www.example.com.cn的解析？

下图是我的当前域名（xiaowu.net）的解析。MX才是和邮件有关的，www是网页的，对吧？烦请帮我看一下下面的解析里哪些是没用的？

![IMG_9047](https://github.com/jeessy2/ddns-go/assets/118618521/693a26a1-8257-48f0-ac26-85700012b94a)

**Reply by jeessy2** [reply]
这个自己根据实际情况看吧。我也不知道哪些能用
